db/migrate/20171002155811_fix_cma_case_first_published_at.rb

```diff
@@ -0,0 +1,15 @@
+class FixCmaCaseFirstPublishedAt < ActiveRecord::Migration[5.1]
+  def change
+    sql = <<-SQL.strip_heredoc
+      UPDATE editions
+      SET    first_published_at = (details->'metadata'->>'opened_date')::date
```

Comment:
I think this needs to cast to a `timestamp` and not `date` to preserve the time. Will update.